components/var_dumper/introduction.rst

```diff
@@ -62,6 +62,17 @@ current PHP SAPI:
     You'll also learn how to change the format or redirect the output to
     wherever you want.
 
+.. tip::
+
+    In order to have the ``dump()`` function always available, you can install
+    it globally in your dev environment:
```

Comment:
It's true that my proposal is more verbose, but I think it explains the purpose of the tip better.

In the original text, what does *"to have the function always available"*? That's why I prefer to say explicitly that this is referring to using it outside Symfony applications.

And the same for `dev environment` or `dev box`. Prefer to say `your machine`, `your computer`, `your local machine`, etc.